index.bs

```diff
@@ -177,6 +177,22 @@ the underlying platform in the next request:
   Sec-CH-UA-Platform: "Windows"
 ```
 
+Infrastructure {#infrastructure}
+==============
+
+This specification depends on Client Hints Infrastructure, HTTP Client Hints, and the Infra
+Standard.
+[[!CLIENT-HINTS-INFRASTRUCTURE]]
+[[!I-D.ietf-httpbis-client-hints]]
+[[!INFRA]]
+
+Some of the terms used in this specification are defined in <cite>Structured Field Values for
+HTTP</cite>.
+[[!I-D.ietf-httpbis-header-structure]]
+
+[=Client Hints Fetch integration=] is defined as part of the [[CLIENT-HINTS-INFRASTRUCTURE]]
```

Comment:
I think if you site the infra document you don't need the specific Fetch bit

(It's also a little weird to bring up the fetch and not the HTML, but that's whatever)

Reply:
Thanks, that's fair.

Can you clarify what you mean by:

> (It's also a little weird to bring up the fetch and not the HTML, but that's whatever)

Are you saying it's weird that there isn't a reference to https://wicg.github.io/client-hints-infrastructure/#html? That's a fair point, I was just moving the existing section 2.8 (https://wicg.github.io/ua-client-hints/#fetch-integration).

Reply:
It was just a comment. Either way, I don't think we need it if just bring up the infra spec as a whole

> (edit: I will never use the word fair again, promise)

Also fair :P